### ISSUE TITLE
All downloaded .zip and .rar files to change to .ck3

### ISSUE THREAD

**Issue by SlothUchiha** [crschnick/pdx_unlimiter#135]
PDX Unlimiter causes the file types of anything downloaded to automatically change. I have winrar set at the default app for .zip and .rar and it happens anyways. I have verified that it only happens because of PDX Unlimiter. I can still right click and then open with another app, however, this is super inconvenient and also causes some other programs to not perform correctly unless I go through and rename each .rar or .zip individually. Windows 11 if it matters.

**Comment by crschnick** [comment]
Yeah that is a known issue, due to xpipe basically being the only program to report supported mime types. You can just use the standalone build for that

**Comment by SlothUchiha** [comment]
Oh ok, thanks for the quick response.
